azure-pipelines.yml

```diff
@@ -88,7 +90,9 @@ stages:
           VS2017:
```

Comment:
On OSX and Linux, we have a separate CI job for each backend. This uses S3 already, so please copy+paste this `VS2017` job to create a new job `VS2017__azure` and replace `TILEDB_S3: ON` with `TILEDB_AZURE: ON`.

The motivation is not obvious, let me explain: our unit tests currently decide which backend to test against, depending on how it's built. In this job, both S3 and Azure are built. The behavior is basically undefined whether this is testing S3 or Azure. This is a known limitation that's on the agenda to fix.

Reply:
Changed.